.github/workflows/amis-workflow.yml

```diff
@@ -0,0 +1,157 @@
+---
+name: amis-workflow
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - 'v*'
+  schedule:
+    - cron: "0 0 * * 0"
+  # Add pull_request for testing
+  pull_request:
+    branches:
+      - master
+jobs:
+  publish-amis-to-marketplace:
+    # needs:
+    #  - release-hold
+    #  - ami-hold
+    if: ${{ needs.path_filter.outputs.should_skip != 'true' }}
+    name: publish-amis-to-marketplace job
+    runs-on: ubuntu-latest
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+      CODE_DIR: "${{ github.workspace }}/experimental/cloudstrapper"
+      VARS_DIR: "${{ github.workspace }}/experimental/cloudstrapper/playbooks/roles/vars"
+      WORK_DIR: "${{ github.workspace }}/experimental/cloudstrapper/playbooks"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run apt
+        run: sudo apt-get update && sudo apt -y upgrade
+      - name: setup pyenv
+        uses: "gabrielfalcao/pyenv-action@v8"
+        with:
+          default: 3.7.0
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+      - name: Install Dependencies
+        run: |
+            pip install ansible awscli boto3
+            sudo apt-get update
+      - name: Determine AWS credentials to use
+        run: |
+            if ${{ github.ref == 'refs/heads/v1.*' && github.event_name != 'schedule' }}; then
```

Comment:
Need to change logic to work with workflow dispatch

Reply:
done